tests/unit/lib/load-dojo-loader.ts

```diff
@@ -52,12 +54,12 @@ registerSuite('lib/load-dojo-loader', {
 
 			assert.equal(configBaseUrl, process.cwd());
 			assert.deepEqual(configPackages, [
-				{ name: 'src', location: '_build/src' },
-				{ name: 'test', location: 'node_modules/test' },
-				{ name: 'dojo-loader', location: 'node_modules/dojo-loader/dist/umd' },
-				{ name: 'rxjs', location: 'node_modules/@reactivex/rxjs/dist/amd' },
-				{ name: 'maquette', location: 'node_modules/maquette/dist' },
-				{ name: 'immutable', location: 'node_modules/immutable/dist' }
+				{ name: 'src', location: path.join('_build', 'src') },
+				{ name: 'test', location: path.join('node_modules', 'test') },
+				{ name: 'dojo-loader', location: path.join('node_modules', 'dojo-loader', 'dist', 'all') },
+				{ name: 'rxjs', location: path.join('node_modules', '@reactivex', 'rxjs', 'dist', 'amd') },
```

Comment:
we should probably raise an issue to drop `rxjs`, `maquette`, and `immutable` at some point...

Reply:
https://github.com/dojo/grunt-dojo2/issues/180